conf/application.conf

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 HM Revenue & Customs
+# Copyright 2021 HM Revenue & Customs
```

Comment:
The controllers section in this file can be removed. Give me a shout if you need more info

Reply:
removed section however not truly sure what it purpose is and why it's okay now to remove